user_manual/files/sharing.rst

```diff
@@ -0,0 +1,115 @@
+============
+File Sharing
+============
+
+Nextcloud users can share files and folders. Possible targets are
+
+* public links
+* users
+* groups
+* circles
+* talk conversations
+* users or groups on federated Nextcloud servers
+
+.. note:: Some options may note be available due to administrative configuration.
+   See :doc:`admin documentation <../../admin_manual/configuration_files/file_sharing_configuration>` for details.
```

Comment:
@nickvergessen Could you please help me with the syntax when cross-referencing between user- and admin manual? I don't seem to get the correct linking.

Reply:
Ah, didn't notice that. Yeah in that case use the full URL you had used before. It will be automatically translated to the version used by the user.